File: VenhaParaOLEDS/Models/Candidato.cs
// Models/Candidato.cs
using System;
using System.Collections.Generic;

namespace VenhaParaOLEDS.Models
{
    public class Candidato
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public DateTime DataNascimento { get; set; }
        public string CPF { get; set; }

        //Relação 1:N com Profissoes
        public List<Profissao> Profissoes { get; set; } = new();
    }
}

File: app/models/candidato.py
#N -> N
from sqlalchemy import Column, String, Date
from sqlalchemy.dialects.postgresql import ARRAY
from ..database import Base

class Profissao(Base):
    __tablename__ = 'profissoes'
    id = Column(String, primary_key=True, index=True)
    nome = Column(String, unique=True, index=True)

class Candidato(Base):
    __tablename__ = 'candidatos'
    cpf = Column(String, primary_key=True, index=True)
    nome = Column(String, index=True)
    data_nascimento = Column(Date)
    profissoes = Column(ARRAY(String))